dashboard/app/controllers/api/v1/pd/applications_controller.rb

```diff
@@ -42,6 +49,14 @@ def update
 
   private
 
+  APPLICATION_TYPES = {
+    csf_facilitators: Pd::Application::Facilitator1819Application.csf,
+    csd_facilitators: Pd::Application::Facilitator1819Application.csd,
+    csp_facilitators: Pd::Application::Facilitator1819Application.csp,
+    csd_teachers: Pd::Application::Teacher1819Application.csd,
+    csp_teachers: Pd::Application::Teacher1819Application.csp,
+  }
```

Comment:
I just realized we can't do it quite this way, because it will remove the `@applications` scope from `load_and_authorize_resource`. It's not relevant now when it's admin only and we can see all applications, but for organizers we will need this to remain scoped. 

We need to do something similar, but on the fly based on `@applications` rather than `Pd::Application::Teacher1819Application`.

I suggest something like making this a function containing a case statement rather than a constant hash.